src/_tutorials/web/fetch-data.md

```diff
@@ -414,7 +414,7 @@ and loads the file.
 
 **Try it!** Click **Run** and then click the **Get portmanteaux** button.
 
-{% comment %} https://gist.github.com/chalin/6b76bce8d46986e624f4e82925c48287 {% endcomment %}
+{% comment %} https://gist.github.com/kwalrath/70bb4db7bea44663206e7a9c111f31d2 {% endcomment %}
 <iframe
 src="{{site.dartpad-embed-html}}?id=6b76bce8d46986e624f4e82925c48287"
```

Comment:
You need to update 6b76bce8d46986e624f4e82925c48287 in this line to be 70bb4db7bea44663206e7a9c111f31d2 instead.

Reply:
Oops Sorry I just saw the comment tag around that. Will update.